sigs.yaml

```diff
@@ -2200,6 +2200,43 @@ workinggroups:
     contact:
       slack: sig-cluster-lifecycle
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle
+  - name: LTS
```

Comment:
There is a `stackholder_sigs` field for Working Groups now, can you add that here as well?

https://github.com/kubernetes/community/blob/ec75f9f55186f47283ce519bad404764e1a71dc6/generator/app.go#L113

Reply:
Will do thanks!!